en/docs/learn/configuring-uniqueness-of-claims.md

```diff
@@ -44,3 +44,5 @@ Follow the steps below to configure WSO2 Identity Server.
 6. Finally, click the `Update` button to save the additional claim property. 
 
 Now you can check and verify the functionality by adding users with existing claim value or update the user claim value of an existing claim.
+If you are successfully configured, you can't create or update users with existing claim value. 
+It will show an error message saying you can not create or update users with existing claim value. 
```

Comment:
Shall we remove this sentence? It looks like a repetition.

Reply:
Removed: abde67e